typescript/sdk/src/utils/sealevel/tokenSerialization.ts

```diff
@@ -0,0 +1,170 @@
+import { PublicKey } from '@solana/web3.js';
+
+import { DomainId } from '@hyperlane-xyz/utils';
+
+/**
+ * Hyperlane Token Borsh Schema
+ */
+export class AccountDataWrapper {
```

Comment:
Is this not duplicated with the existing code in sdk/src/utils/sealevel.ts?

Reply:
Oh my mistake, that's not on main yet. It's here: https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/2559/files#diff-7634ab7e00eb72d84a0cc53d6d4f78c266d72f1832d2db3473a30d94f06eeac4

Reply:
I'll let you know if my PR merges first. Otherwise go ahead and I'll de-dupe